docs/modules/operation/pages/deep-dive/admin/restart.adoc

```diff
@@ -4,14 +4,14 @@
 This section provides an example of the order to follow when shutting down and restarting a sample {page-component-title} system: {page-component-title} core layer, persistence and messaging layer, and database layer.
 
 .{page-component-title} restart process
-image::admin/restart-process.png["Diagram displaying {page-component-title} components and the order in which to restart them"]
+image::admin/restart-process.png["Diagram displaying {page-component-title} components and the order in which to restart them."]
 
 Note that restarting the whole stack is rare.
-Usually, restarting the {page-component-title} core happens for reconfiguration or to run upgrades.
-A core restart takes between 2-5 minutes depending on memory configuration.
+Usually, the {page-component-title} core is restarted only to apply configuration changes, or to run upgrades.
+A core restart takes between 2-5 minutes, depending on memory configuration.
```

Comment:
"A Core restart can take upwards of 5 minutes depending on the configuration but usually takes less time."?

Reply:
@Bonrob2, what do you think about rephrasing this? I think that "between 2-5 minutes" works well, but there are probably edge cases where a restart takes more time.

Reply:
Can we just not reference the amount of time for a restart?

Reply:
Let's compromise, and put the positive first:

"A core restart usually takes less than 5 minutes, but can take longer depending on the configuration."